### DISCUSSION TITLE
Unexplainable <p>...</p> in feather

### DISCUSSION THREAD

**Discussion by eawmaas** [xenocrat/chyrp-lite#204] [Q&A] [answered]
In quote.twig there´s: {{ post.source }}
This generates a `<p></p>`-pair around the text.

Where does that come from? I want to get rid of the `<p></p>` but how?

**Comment by xenocrat** [comment, marked as answer]
Do you have Markdown enabled in the content settings? Paragraphs are [the default block container](https://github.com/xenocrat/chyrp-lite/wiki/Chyrp-Flavoured-Markdown#paragraphs) in Markdown.

**Reply by eawmaas** [reply]
I see, yes. When I turn it off the tags are gone.
I was trying make a feather and give a header to <section> as the html5-specs require and then the tags popped up out of nowhere, been trying to find their origin all day yesterday....

Thank you, I know now that I need to rethink this.
